### DISCUSSION TITLE
Remount folder has no file under top directory

### DISCUSSION THREAD

**Discussion by summerisc** [seaweedfs/seaweedfs#1919] [Q&A] [answered]
I fuse mount a folder to a local path, stops the weed mount, run the exact same weed mount cmd again to remount the same thing, after that, I can only see the top-level folder, but all the subdirectories and files under these folders not showing on the local path. I confirm in the filer that the data are all there, it's just not show up after the remount.

cmd I use:
1. `sudo ./weed mount -filer=10.244.2.191:8888 -dir=/home/usr/test -filer.path=/test`
2. terminate this cmd
3. run `sudo ./weed mount -filer=10.244.2.191:8888 -dir=/home/usr/test -filer.path=/test` again.

I am using seaweedfs run on k8s and expose filer with a local host if that makes a difference

**Comment by chrislusf** [comment, marked as answer]
Make sure you are using version 2.34

**Reply by summerisc** [reply]
lol thanks, I saw 2.32 just released a few days ago so I used that

**Reply by summerisc** [reply]
when I upgraded to 2.34 this problem still exists, any thoughts?

**Reply by chrislusf** [reply]
can not tell with your description. need to make a reproducible case.

**Reply by summerisc** [reply]
sure, let me know if this is the clear steps to reproduce:
1. using 2.34 version of image
2. create a folder with the address of /home/usr/test 
3. run `sudo ./weed mount -filer=10.244.2.191:8888 -dir=/home/usr/test -filer.path=/test` to mount this path to /test folder under weed
4. create a folder under /test like /test/foobar1, copy some files from local machine to /test/foobar1 (this is where I found the speed is low, but that's another topic)
5. use ctrl + c to terminate the command, mount disappears
6. run the same command in step 3 again to remount
7. found foorbar1 folder still there, but cannot see all the files copy to this foobar1, open foobar1 get an empty folder
8. check the data in the filer, all good, the data under /test/foobar1 still exists in filer.
Let me know if I could provide anything else, or I might be wrong using the fuse feature somewhere. Appreciate it.

**Reply by chrislusf** [reply]
can not reproduce.

```
$ pwd
/Users/chris/tmp/m2
m2$ cd foobar1/
foobar1$ time cp ~/Downloads/Element.dmg .
cp: /Users/chris/Downloads/Element.dmg: could not copy extended attributes to ./Element.dmg: Operation not permitted

real	0m0.532s
user	0m0.001s
sys	0m0.180s
foobar1$ cd


# restart mount on another terminal

~$ cd -
/Users/chris/tmp/m2/foobar1
foobar1$ ls -al
total 220768
drwxr-xr-x  1 chris  staff          0 Mar 17 18:15 .
drwxr-xr-x  1 chris  staff          0 Mar 17 18:19 ..
-rw-r--r--  1 chris  staff  113033025 Mar 17 18:18 Element.dmg
```

**Reply by chrislusf** [reply]
Try to make a docker compose file, and see whether it is reproducible.

**Reply by summerisc** [reply]
close this due to figure out the root cause, used to be using hostPath for MySQL storage, changed to local storage then it works
